docs/CONTRIBUTING.md

```diff
@@ -1,3 +1,7 @@
+<!--
+Copyright (c) ONNX Project Contributors
+-->
+
```

Comment:
Duplicate `<!--Copyright (c) ONNX Project Contributors-->`?

Reply:
Thank you! 

Yes, I removed it ["reuse lint" did not realize the information was there, only combined with copyright and license]. I will check that, how to handle that, if we want to check such cases automatically by a tool.